Pipfile

```diff
@@ -8,6 +8,7 @@ click = "*"
 inquirer = "*"
 gitpython = "*"
 colorama = "*"
+pytest = "*"
```

Comment:
Shouldn't there be a Pipfile.lock update, as well? Can you commit that?

Reply:
I took care of the Pipfile changes in my previous PR, so this is on master already. You should remove the `Pipfile` and `Pipfile.lock` files from your commit (if they differ from the ones I committed. If they're the same, git will just take care of it.)